tests/integration/helm_fluentbit_fluentd_test.go

```diff
@@ -327,9 +325,9 @@ func Test_Helm_FluentBit_Fluentd(t *testing.T) {
 			10, // we don't really control this, just want to check if the logs show up
 			map[string]string{
 				"cluster":         "kubernetes",
-				"_sourceName":     "",
+				"_sourceName":     "(?!undefined$).*",
```

Comment:
I guess that since we started adding constants for regexes, this one could get moved to a constant, too.